libs/barista-components/sunburst/src/sunburst.util.ts

```diff
@@ -0,0 +1,466 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DtColors,
+  DT_CHART_COLOR_PALETTE_ORDERED,
+} from '@dynatrace/barista-components/theming';
+import * as d3 from 'd3-shape';
+
+export interface DtSunburstNode {
+  label: string;
+  value?: number;
+  color?: DtColors | string;
+  children?: DtSunburstNode[];
+}
+
+export enum DtSunburstValueMode {
+  ABSOLUTE = 'absolute',
+  PERCENT = 'percent',
+}
+
+export interface DtSunburstNodeInternal {
+  origin: DtSunburstNode;
+
+  id: string;
+  label: string;
+  value: number;
+  valueRelative: number;
+  children?: DtSunburstNodeInternal[];
+
+  depth: number;
+  color: DtColors | string;
+  colorHover: DtColors | string;
+  isCurrent: boolean;
+  visible: boolean;
+  active: boolean;
+  showLabel: boolean;
+}
+
+export interface DtSunburstSlice
+  extends d3.PieArcDatum<DtSunburstNodeInternal> {
+  path: string | null;
+  labelPosition: [number, number];
+  tooltipPosition: [number, number];
+  showLabel: boolean;
+}
+
+const SVG_SETTINGS = {
+  outerRadius: 160,
+  innerRadius: 64,
+  labelOffsetRadius: 32,
+  tooltipOffsetRadius: 64,
+  minAngleForLabel: (15 / 360) * 2 * Math.PI,
```

Comment:
This seems like a very curious calculation. Could you please add a comment above, what this calculation does. 
Something like: 
```suggestion
  // Minimum angle for the label should be 15°, needs to be converted into radians for further processing.
  minAngleForLabel: (15 / 360) * 2 * Math.PI,
```